scripts/build/genFaultTable.py

```diff
@@ -115,7 +115,6 @@ def main():
     file.close()
 
     faults_config_file_name = sys.argv[2] + "/faults.config"
-    faults_test_config_file_name = sys.argv[2] + "tests/faults.config"
```

Comment:
I changed the astrobee/config/tests/faults.config to be a simlink of the actual astrobee/config/faults.config, therefore no need to generate it. This was done to minimize error if someone manually changes this file for testing.